tensorflow/lite/kernels/detection_postprocess.cc

```diff
@@ -498,7 +498,7 @@ TfLiteStatus NonMaxSuppressionMultiClassRegularHelper(TfLiteContext* context,
 
   int size_of_sorted_indices = 0;
   std::vector<int> sorted_indices;
-  sorted_indices.resize(max_detections);
+  sorted_indices.resize(num_boxes + max_detections);
   std::vector<float> sorted_values;
   sorted_values.resize(max_detections);
```

Comment:
What about sorted_values? Wouldn't it have the same issue?

Reply:
@jdduke Thank you for your reviewing.

I don't think sorted_values has the same issue.

The access range of sorted_values is [0, **num_indices_to_sort**).
```
for (int row = 0; row < num_indices_to_sort; row++) {
  sorted_values[row] = ....
}
```

Where **num_indices_to_sort** is garanteed that smaller than max_detections.

```
int num_indices_to_sort = std::min(output_index, max_detections);
```

Reply:
That sounds right. I wonder if there's a simple test case we can add which exercises this? If not, no worries.

Reply:
@jdduke Thank you for your reviewing.

I also think it would be better if there is a test.
But, I'm afraid that I can't add the test case because I'm not familiar with the test framework.